code/datums/ductnet.dm

```diff
@@ -1,23 +1,30 @@
 ///We handle the unity part of plumbing. We track who is connected to who.
 /datum/ductnet
+	///Stuff that can supply chems
 	var/list/suppliers = list()
+	////Stuff that can take chems
 	var/list/demanders = list()
+	///All the ducts that make this network
 	var/list/obj/machinery/duct/ducts = list()
 
+	///Max reagents we can carry per tick
 	var/capacity
+
 ///Add a duct to our network
 /datum/ductnet/proc/add_duct(obj/machinery/duct/D)
 	if(!D || (D in ducts))
 		return
 	ducts += D
 	D.duct = src
+
 ///Remove a duct from our network and commit suicide, because this is probably easier than to check who that duct was connected to and what part of us was lost
 /datum/ductnet/proc/remove_duct(obj/machinery/duct/ducting)
 	destroy_network(FALSE)
 	for(var/obj/machinery/duct/D in ducting.neighbours)
 		addtimer(CALLBACK(D, /obj/machinery/duct/proc/reconnect), 0) //all needs to happen after the original duct that was destroyed finishes destroying itself
```

Comment:
This proc doesn't exist anymore, can you fix that?

Reply:
No idea how I didnt notice this. It's gone now